packages/common/src/api/tan-query/jupiter/utils.ts

```diff
@@ -0,0 +1,174 @@
+import { AudiusSdk } from '@audius/sdk'
+import {
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  createAssociatedTokenAccountIdempotentInstruction,
+  createCloseAccountInstruction,
+  createTransferCheckedInstruction,
+  getAccount,
+  getAssociatedTokenAddressSync
+} from '@solana/spl-token'
+import { PublicKey, TransactionInstruction } from '@solana/web3.js'
+
+import { JUPITER_PROGRAM_ID } from './constants'
+import { UserBankManagedTokenInfo } from './types'
+
+export async function addUserBankToAtaInstructions({
```

Comment:
nit: tbh not sure if we need to have helpers for this and `addAtaToUserBankInstructions` - the function bodies are nearly smaller than the parameter list, probably easier to inline